microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -191,43 +227,62 @@ To submit the YouTube video URL of your video:
 1. On the store listing details webpage, paste the URL in the **YouTube video URL** field.
 
 
+<!-- ------------------------------ -->
 #### Search terms requirements
 
+<!-- todo: webpage, or section of webpage (which webpage)? -->
+
 Search terms must meet the following requirements:
 
 *  You can enter search terms to use up to a maximum of 21 words.  Whether used as single words, phrases, or a combination of both, you're only allowed a maximum of 21 words.
 
 *  Up to a maximum of seven search terms (single word or phrases).  Each search term has a character limit of 30 characters.
 
 
+<!-- todo: what UI actions to fill in this webpage and proceed?
+
+1. Click the **Foo** button.  The **Submit your extension** webpage opens.
+-->
+
+
 <!-- ========================================================================== -->
-## Step 6: Complete your submission
+## Step 6: Complete the submission by providing testing notes
 
 On the **Submit your extension** webpage, add notes for certification to help test your extension.
 
 
+<!-- ------------------------------ -->
 #### Notes for certification (optional)
 
+<!-- todo: how do you get from the **Submit your extension** webpage to the **Notes for certification** webpage? -->
```

Comment:
how do you get from the **Submit your extension** webpage to the **Notes for certification** webpage?